vcs/util/path.go

```diff
@@ -0,0 +1,10 @@
+package util
+
+// Rel strips the leading "/" prefix from the path string, effectively turning
+// an absolute path into one relative to the root directory.
+func Rel(path string) string {
+	if len(path) > 0 && path[0] == '/' {
+		return path[1:]
+	}
+	return path
```

Comment:
>  I think you should just use strings.TrimPrefix instead for better readability.

Yep, good point. I've pushed a commit to address this just now.

> Additionally, are you sure doing the above is the right thing? What if the input is "/", then the output is a blank string. Would that work?

In all of my testing a blank string does work, but my testing didn't include Lstat which it looks like [has a special case in treating "." as "root directory"](https://sourcegraph.com/sourcegraph.com/sourcegraph/go-vcs@72633312692c1d28164bda8da8dc3b814a24b0aa/.tree/vcs/gitcmd/repo.go#startline=891&endline=892), so I'll add a special case for "/".